fixtures/genesis-sectors/README.md

```diff
@@ -0,0 +1,26 @@
+## Generating genesis-sectors
+
+Genesis sectors are generated using the [Lotus](https://docs.lotu.sh/en+setup-local-dev-net) seed tool.
+Follow the directions there to install lotus and then run:
+```
+./lotus-seed pre-seal --sector-size 2048 --num-sectors 2 --miner-addr=t0106
+```
+The sector-size must be one the sizes in use by the proof system. And the miner address should be the address
+of the bootstrap miner to be created with gengen. This will create a `.genesis-sectors` directory in your home directory.
```

Comment:
Might be nice to leave a hint that the inlined `t0106` is right: "by default in gengen this address starts at `t0106` and the id address number is incremented for every additional miner"